src/wwwroot/controllers/PlanCtrl.js

```diff
@@ -35,7 +35,8 @@
     vm.planInfoLoader = true;
     vm.planStatusInfoLoader = true;
     vm.ipsPlanCount = 0;
-
+    vm.errorsPlanUpgrade = UpgradeableStatusPlan();
+    vm.isValidUpgradeablePlan = isValidUpgradeablePlan();
```

Comment:
It is evaluating once when this code is executed, so you are loosing the databinding, is it fine?